src/tests/ftest/cart/test_group_np_cli.c

```diff
@@ -138,14 +140,25 @@ test_run(void)
 
 			for (tag = 0; tag < test_g.t_srv_ctx_num; tag++) {
 				DBG_PRINT("Sending rpc to %d:%d\n", rank, tag);
-				check_in(grp, rank, tag);
+
+				k = test_g.t_num_checkins_to_send;
+				for (j = 0; j < k; j++) {
+					check_in_with_delay(grp,
+							    rank,
+							    tag);
+				}
 			}
 		}
 
+		/* Don't wait on the aborted rank */
+		icea = test_g.t_issue_crt_ep_abort;
 		for (i = 0; i < rank_list->rl_nr; i++) {
-			for (tag = 0; tag < test_g.t_srv_ctx_num; tag++) {
-				tc_sem_timedwait(&test_g.t_token_to_proceed, 61,
-						 __LINE__);
+			rank = rank_list->rl_ranks[i];
+			if (icea != rank) {
+				for (tag = 0; tag < test_g.t_srv_ctx_num; tag++) {
```

Comment:
(style)  line over 80 characters
(style)  suspect code indent for conditional statements (32, 48)